CMakeLists.txt

```diff
@@ -0,0 +1,99 @@
+cmake_minimum_required(VERSION 3.15)
+cmake_policy(SET CMP0091 NEW) # Use MSVC_RUNTIME_LIBRARY to select the runtime
+project(Verilator
+    VERSION 4.025.99
+    HOMEPAGE_URL https://verilator.org
+    LANGUAGES CXX
+)
+
+option(CMAKE_UNITY_BUILD
+    "Builds concatenated source files, potentially sppeding up the build.")
+option(DEBUG_AND_RELEASE_AND_COVERAGE
+    "Builds both the debug and release binaries, overriding CMAKE_BUILD_TYPE. Not supported under MSBuild.")
+
+set(CMAKE_INSTALL_DATADIR ${CMAKE_INSTALL_PREFIX})
+include(GNUInstallDirs)
+include(CMakePackageConfigHelpers)
+include(CheckStructHasMember)
+include(ExternalProject)
+
+if (MSVC)
```

Comment:
Please add something to make it clear that this cmake is currently only supported on MSVC, not Linux